settings.gradle

```
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "MiniGH_Base"
```

Just calling it MiniGH is fine I think

```suggestion
rootProject.name = "MiniGH"
```